package.json

```diff
@@ -52,7 +53,7 @@
     "watchify": "^3.7.0"
   },
   "dependencies": {
-    "popsicle": "^8.2.0",
+    "any-promise": "^1.3.0",
```

Comment:
I understand your pain with configuring Promises. But in real projects this should be done globally only once, since Promise is a part of standard. There's no need for extra library for that.